default.properties

```diff
@@ -137,4 +147,4 @@ smtpPort=${SMTPPORT}
 # console will add all links to analytics and also execute XHR requests
 # header will diplay analytics app
 # default: true
-# analyticsEnabled=true
+analyticsEnabled=false
```

Comment:
Has this been changed on purpose (as it is still `true` in the comment)?

Reply:
Oups